example/tables.jl

```diff
@@ -0,0 +1,28 @@
+# This file is essentially the same as test/tables.jl
+# Watch for differences in future releases.
+
+using DataFrames, LaTeXTabulars, Random, SolverBenchmark
```

Comment:
I think (here and elsewhere), we should separate `using` statements with standard library packages from those with external packages for clarity. What do you think? We should probably also identify them:
```julia
# stdlib imports
using Logging, Printf  # in alphabetical order

# dependencies imports
using DataFrame, LaTeXTabulars  # in alphabetical order

# JSO imports
using Krylov, LinearOperators  # in alphabetical order
```
What do you think?

Reply:
That's a good guideline